app/_hub/kong-inc/key-auth/index.md

```diff
@@ -226,6 +226,7 @@ In both cases, the fields/parameters work as follows:
 field/parameter     | description
 ---                 | ---
 `{consumer}`        | The `id` or `username` property of the [Consumer][consumer-object] entity to associate the credentials to.
+`ttl`               | The number of seconds the key is going to be valid.
```

Comment:
```suggestion
`ttl`<br>*optional*               | The number of seconds the key is going to be valid. If missing the ttl is unlimited.
```

Reply:
Thank you for the PR. I edited the suggestion and have requested review from core team for technical accuracy. cc @kikito @gszr

Reply:
Direct review link:  https://deploy-preview-2873--kongdocs.netlify.app/hub/kong-inc/key-auth/#create-a-key

Also, the edit does not seem to appear in the build at this time. Perhaps the formatting is off.

![image](https://user-images.githubusercontent.com/3756245/118973404-42345d80-b937-11eb-84f4-533100002e14.png)

This PR is an edit to the documentation. The description says the feature is not documented, so this is an improvement to the docs.

Reply:
I reran the build but still the edit does not appear.